apps/docs/features/docs/GuidesMdx.template.tsx

```diff
@@ -120,6 +121,7 @@ const GuideTemplate = ({ meta, content, children, editLink, mdxOptions }: GuideT
           hideToc ? 'col-span-12' : 'col-span-12 md:col-span-9'
         )}
       >
+        <Breadcrumbs className="mb-3 w-full" />
```

Comment:
Do you think the spacing on the bottom is a little tight? Your final call on this (you're the expert 😄)

<img width="869" alt="image" src="https://github.com/supabase/supabase/assets/26616127/4ebfe4be-4b0f-498c-a4b0-260958047aab">

Reply:
you're right, mb-2 is better 😁